specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimanagement.json

```diff
@@ -0,0 +1,484 @@
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "paths": {
+    "/policies": {
+      "get": {
+        "tags": [
+          "Policy"
+        ],
+        "operationId": "Policy_List",
+        "description": "Lists all the Global Policy definitions of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementListPolicies": {
+            "$ref": "./examples/ApiManagementListPolicies.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Policy scope.",
+            "enum": [
+              "Tenant",
+              "Product",
+              "Api",
+              "Operation",
+              "All"
+            ],
+            "x-ms-enum": {
+              "name": "PolicyScopeContract",
+              "modelAsString": false
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns an array of Policy Contracts.",
+            "schema": {
+              "$ref": "#/definitions/PolicyCollection"
+            }
+          }
+        }
+      }
+    },
+    "/policies/{policyId}": {
+      "get": {
+        "tags": [
+          "Policy"
+        ],
+        "operationId": "Policy_Get",
+        "description": "Get the Global policy definition of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementGetPolicies": {
+            "$ref": "./examples/ApiManagementGetPolicies.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PolicyIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the Policy Contracts.",
+            "schema": {
+              "$ref": "#/definitions/PolicyContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Policy"
+        ],
+        "operationId": "Policy_CreateOrUpdate",
```

Comment:
>Policy_CreateOrUpdate [](start = 24, length = 21)

Should this be marked as long running operation?

Reply:
no.. except the apimtenant.json operation, none are Long Running.

---
In reply to: [163336898](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163336898) [](ancestors = 163336898)

Reply:
you can check the response code, 201, not 202

---
In reply to: [163337706](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163337706) [](ancestors = 163337706,163336898)